### PR TITLE
add thermal test case based on SPE9_CP

### PR DESCRIPTION
**Pull request by andlaus**
this adds a simple thermal deck that is based on SPE9_CP.

### CODE REVIEW

**Comment by atgeirr** [comment]
If I want to test it, what (if any) branches do I have to use of ewoms or opm-simulators?

**Comment by andlaus** [comment]
opm-common: master
opm-material: master
ewoms: master
opm-simulators: @totto82's blackoil_energy_WIP. this one needs to be rebased on top of the current master in order to compile and it runs into an assertation within the well model, so your mileage may vary.

**Comment by andlaus** [comment]
(of course using E300 should work as well, although I obviously cannot test this.)

**Comment by akva2** [comment]
if this is meant as the basis of a regression test, opm-data is not the right repo.

**Comment by andlaus** [comment]
So far this is more meant as another example of how to use the thermal capability. once the well model is ready, I'll add something to opm-tests and opm-simulators.

File: spe9/SPE9_CP_THERMAL.DATA [hunk 1]
```diff
@@ -0,0 +1,571 @@
...
+	600*0.157 /
+
+-- PERMX, PERMY & PERMZ
+INCLUDE
+	PERMVALUES.DATA /
+
+-- total thermal conductivty
+THCONR
+       300*30.0 / -- btu/(day / ft / °F)
+
```
Comment (totto82) [inline spe9/SPE9_CP_THERMAL.DATA:122]: c/p error from SPE1? SPE9 has 9000 cells

Comment (andlaus) [reply spe9/SPE9_CP_THERMAL.DATA:122]: err, yes; that was embarrassing. fixed & thanks. strangely enough, the thermal WIP branch for flow worked for it. I should probably find out why...

BTW: I just noticed that CP-grid could be short for "copy-and-paste" grid. That would probably be not too far from the truth in many instances ;)

**Comment by andlaus** [comment]
opm-data is not used for regression testing anymore. closing.

**Pull request closed without merging**